nixops/backends/__init__.py

```diff
@@ -247,16 +247,29 @@ def reboot_sync(self, hard: bool = False) -> None:
         """Reboot this machine and wait until it's up again."""
         self.reboot(hard=hard)
         self.log_start("waiting for the machine to finish rebooting...")
-        nixops.util.wait_for_tcp_port(
-            self.get_ssh_name(),
-            self.ssh_port,
-            open=False,
-            callback=lambda: self.log_continue("."),
```

Comment:
Can we get the `.`s back? I think they are pretty important from a "is nixops even still trying, or is it stuck?" perspective.

Reply:
Can there be a real spinner as opposed to just dots? A quick google points me to https://github.com/manrajgrover/py-spinners and https://github.com/verigak/progress/

Reply:
For now I just re-added the dots. I'd love to make the output more pretty, but that's a separate PR.